wled00/e131.cpp

```diff
@@ -194,35 +194,36 @@ void handleE131Packet(e131_packet_t* p, IPAddress clientIP, byte protocol){
             return;
 
           if (e131_data[dataOffset+1] < strip.getModeCount())
-            effectCurrent = e131_data[dataOffset+ 1];
-          effectSpeed     = e131_data[dataOffset+ 2]; // flickers
-          effectIntensity = e131_data[dataOffset+ 3];
-          effectPalette   = e131_data[dataOffset+ 4];
-          col[0]          = e131_data[dataOffset+ 5];
-          col[1]          = e131_data[dataOffset+ 6];
-          col[2]          = e131_data[dataOffset+ 7];
-          colSec[0]       = e131_data[dataOffset+ 8];
-          colSec[1]       = e131_data[dataOffset+ 9];
-          colSec[2]       = e131_data[dataOffset+10];
+            if (e131_data[dataOffset+1] != seg.mode)  {strip.setMode(id,  e131_data[dataOffset+1]); stateChanged = true;}
+          if (e131_data[dataOffset+2]   != seg.speed)     {seg.speed     = e131_data[dataOffset+2]; stateChanged = true;}      
+          if (e131_data[dataOffset+3]   != seg.intensity) {seg.intensity = e131_data[dataOffset+3]; stateChanged = true;}
+          if (e131_data[dataOffset+4]   != seg.palette)   {seg.palette   = e131_data[dataOffset+4]; stateChanged = true;}
```

Comment:
You can use `seg.setPalette()` to enable palette transitions.

Reply:
There are also some other segment methods like `seg.setMode(), seg.setOpacity(), ..`. Due to transition triggers, to me it's unclear what is the better approach here. But I'll test what feels better/smoother when sending DMX. I'll leave a comment on these tests afterwards to decide ..